### ISSUE TITLE
LightGBM

### ISSUE THREAD

**Issue by cnuernber** [tech-ascent/tech.ml-base#4]
https://github.com/Microsoft/LightGBM

A gradient boosted tree. Similar to xgboost.

**Comment by cnuernber** [comment]
Not necessary for now
